src/components/styles/Theme/ThemeProvider.tsx

```diff
@@ -281,14 +297,10 @@ export interface TypographyTheme {
   };
   weights: {
     regular: number;
+    medium?: number;
```

Comment:
is it safer to have those required?

Reply:
We don't have medium in zopa theme, but  happy to add it and set it to 500 if it makes more sense?

Reply:
Updated. Also updated bold to requried as well.